add-products/products.js

```diff
@@ -0,0 +1,39 @@
+import renderSpeakers from './render-speakers.js';
+import { getProducts, toUSD } from '../common/utils.js';
+
+const speakers = getProducts();
+const list = document.getElementById('speakers');
+
+const button = document.querySelector('button');
+
+button.addEventListener('click', (eventrefresh) => {
+    eventrefresh.preventDefault();
+    const form = document.querySelector('form');
+    const data = new FormData(form);
+
+    const numberizedPrice = Number(data.get('price'));
+    const newSpeaker = {
+        name: data.get('name'),
+        id: data.get('id'),
+        price: toUSD(numberizedPrice),
```

Comment:
The `toUSD` function is unnecessary here. Remember, the price value should be a number in the speaker object, if you add the $ sign it's going to come out as a string.